Refactor Bugzilla link building, add link to opt-in to all missing pages

flodolo opened the pull request.

pascalchevrel commented on views/optin.inc.php:
```diff
@@ -40,6 +40,36 @@
+                '&target_milestone=---&version=Development%2FStaging' .
+                '&format=__default__&cf_locale=' . $bugzilla_locale_name;
+
+    return $bug_link;
```
Since we also create bugzilla links in views/listsitesforlocale.inc.php do you think it would make sense to make a proper function ?

flodolo replied:
Yes, I think it makes definitely sense.

flodolo commented:
Code updated and moved to a Bugzilla class extendind Bugzilla/Bugzilla (will need to run composer update to update autload).

Notes:
* Removed the different qa_contact for Persian (never seen a bug filed for Persian), all bug have me as assignee and you as QA contact.
* Component name is determined outside of the cycle because we need to look for it only once, not for every single link (especially listsitesforlocale).

pascalchevrel commented on classes/Langchecker/Bugzilla.php:
```diff
@@ -0,0 +1,63 @@
+<?php
+namespace Langchecker;
+use Bugzilla\Bugzilla as _Bugzilla;
```
nit: blank line between namespace and use statement

pascalchevrel commented on classes/Langchecker/Bugzilla.php:
```diff
@@ -0,0 +1,63 @@
+     * @return  string                    URL to file the bug on Bugzilla
+     */
+
+    public static function getNewBugLink ($locale, $bugzilla_locale, $bug_type, $files)
```
nit, remove space before parenthesis

flodolo replied:
I put that on purpose, didn't realize it's needed only for closures :-1:

pascalchevrel commented:
LGTM

flodolo commented:
Leaving the merge to you. Not sure if the cronjob on the server runs composer update when pulling.
